resources.nix

```diff
@@ -59,6 +59,8 @@ let
           gomod2nix
           go-modules-sync
           rnix-lsp
+          pass
+          gnupg
```

Comment:
This can be useful for generating keys for gaia, not sure if I should keep it around or not though....